testPI2/unit/shared/setup.test.js

```diff
@@ -37,7 +37,7 @@ Test('setup', setupTest => {
     sandbox.stub(RequestLogger, 'logRequest')
     sandbox.stub(RequestLogger, 'logResponse')
 
-    Sidecar.connect = sandbox.stub()
+    sandbox.stub(Sidecar, 'connect')
```

Comment:
why did you change this from Sidecar.connect = sandbox.stub()?

Reply:
This was causing the tests to fail in admin/participants/handler.test.js .. since the Sidecar.connect was not being restored